src/components/ActionCards/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { actionCardContent } from '../../constants';
+import './actionCards.css';
+
+const renderIcon = Icon => (
+  <Icon size={50} color="grey" style={{ paddingRight: '10px' }} />
+);
+
+const ActionCards = () => (
+  <div className="card-wrapper">
+    {actionCardContent.map(item => (
```

Comment:
Do you think we should inject the items to be rendered to this component? That way if we write tests in the future for this we can inject things to test rendering as opposed to testing the constants data.